avocado/utils/cpu.py

```diff
@@ -239,7 +239,7 @@ def set_cpuidle_state(state_number="all", disable=1, setstate=None):
             for state_no in states:
                 state_file = "/sys/devices/system/cpu/cpu%s/cpuidle/state%s/disable" % (cpu, state_no)
                 try:
-                    open(state_file, "w").write(str(disable))
+                    open(state_file, "wb").write(bytes(disable))
```

Comment:
This breaks badly on py3 as on py3 `bytes(int)` simply initializes so many `\x00` bytes.

Reply:
What do you mean by outdated. It introduces a breakage which is fixed in the next commit. Due to fact that in this revision py3 check is disabled I'm not completely against, but it is valid.

Reply:
I mean that change is not in the branch associated with this PR, it's an outdated commit.

Reply:
Oh, you're right about it.  Now I see it on edccbf330dcd27f2a6178f75c7c080d36dc34fcc.